config/triplea/schemas/lobby_server.json

```diff
@@ -32,9 +32,7 @@
         "description": "The http lobby server host URI",
         "type": "string",
         "anyOf": [
-          { "format": "host-name" },
-          { "format": "ipv4" },
-          { "format": "ipv6" }
+          { "format": "uri" }
```

Comment:
If there's only one thing left in anyOf, you might as well set it directly.

Reply:
Simplified in: 48f0bdd